addons/mail/models/discuss/discuss_channel.py

```diff
@@ -659,6 +659,9 @@ def _message_post_after_hook(self, message, msg_vals):
         Automatically set the message posted by the current user as seen for themselves.
         """
         self._set_last_seen_message(message, notify=False)
```

Comment:
(another PR too) Can't we remove this `notify` False and then remove some adjustment code from the JS part (probably in message post and/or new message handler) if the python properly sends the information?